packages/x-adapter/src/mappers/combine-mappers.ts

```diff
@@ -0,0 +1,29 @@
+import { deepMerge } from '@empathyco/x-deep-merge';
+import { DeepPartial } from '@empathyco/x-utils';
+import { Mapper } from '../types/mapper.types';
+
+/**
+ * Combines the given {@link Mapper | mappers} into a single {@link Mapper | mapper function}.
+ *
+ * @param mappers - The {@link Mapper | mapper functions} to combine.
+ * @returns The combined {@link Mapper | mapper function}.
+ *
+ * @public
+ */
+export function combineMappers<From, To>(
+  ...mappers: Mapper<From, DeepPartial<To>>[]
+): Mapper<From, To> {
+  return mappers.length === 1
+    ? (mappers[0] as Mapper<From, To>)
+    : (from, context) => {
+        if (!context.to) {
+          context.to = {};
+        }
+
+        return mappers.reduce<DeepPartial<To>>((result, mapper) => {
```

Comment:
`result` may be confusing (we have a model with the same name). Maybe change to:
`value` and `mappedValue`